src/scripts/pokemons/PokemonList.ts

```diff
@@ -1795,7 +1795,7 @@ const pokemonList = [
         "name": "Eevee",
         "catchRate": 45,
         "evolution": "Vaporeon, Jolteon, Flareon, Espeon, Umbreon",
-        "evoLevel": "Water Stone, Thunder Stone, Fire Stone, Time Stone, Time Stone",
+        "evoLevel": "Water_stone, Thunder_stone, Fire_stone, Time Stone, Time Stone",
```

Comment:
For consistency:
```suggestion
        "evoLevel": "Water_stone, Thunder_stone, Fire_stone, Time_Stone, Time_Stone",
```

Reply:
Also on a side note, Would it be better to store these as an Array:
```javascript
"evoLevel": ["Water_stone", "Thunder_stone", "Fire_stone", "Time_Stone", "Time_Stone"],
```

Or even something like this:
```javascript
"evoLevel": {
  "Water_stone": ["Vaporeon"],
  "Thunder_stone": ["Jolteon"],
  "Fire_stone": ["Flareon"],
  "Time_Stone": ["Espeon", "Umbreon"],
},
```